docs/import/manage_layers.html

```diff
@@ -73,6 +73,17 @@
   </div>
   <p>Upload additional assets via the <a href="https://code.earthengine.google.com/" target="_blank"> code editor </a></p>
   <br>
+  <div>
+    <div>Add Other Layer Type</div>
+    <select id="non-eelayer-type">
+      <option value="4">Map Tiles</option>
+      <option value="5">KML</option>
+    </select>
+    <label>Layer Name</label><input id="non-eelayer-name"></input>
```

Comment:
When we add ee layers we initialize the layer with an empty name. Might be nice to remove this input so it's parallel to what happens with ee assets (add layer -> add name in Layer Name cell)